code/src/Components/Footer/Footer.css

```diff
@@ -0,0 +1,64 @@
+.footer-box {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    width: 100%;
+    max-width: 270px;;
```

Comment:
In the CSS file Footer.css, there's a typo in the max-width property. Remove the extra semicolon after the value.